### PR TITLE
make tools title translateble

### PR DESCRIPTION
**Pull request by flame51**

### CODE REVIEW

File: pyzo/tools/__init__.py [hunk 1]
```diff
@@ -210,7 +210,7 @@ def loadToolInfo(self):
                     if i<0: continue
                     line = line.rstrip("\n").rstrip("\r")
                     line = line[i+1:].strip(" ")
-                    toolName = line.strip("'").strip('"')
+                    toolName = eval(line)
```
Comment (almarklein) [inline pyzo/tools/__init__.py]: I really don't like the use of eval unless in places where it makes sense :D

**Comment by almarklein** [comment]
Thanks for this. Though I recall that there was a reason why these were not translated. I'll have to look into it better to find out what that was about. I hope to find time soon to do that, but now I'm at a conference.

**Comment by flame51** [comment]
Thanks. I hope all the things on the GUI could be translateble.

File: pyzo/tools/__init__.py (hunk 1 above)
Comment (almarklein) [reply pyzo/tools/__init__.py]: aah, now I realize that this is what makes the other changes possible. Not a big fan of `eval()` but I guess its use is warranted here ...

**Comment by almarklein** [comment]
Thanks!

**Comment by flame51** [comment]
glad to be helpful. :)

**Pull request merged**